register-computer/main.py

```diff
@@ -98,6 +102,47 @@ def __connect_to_activedirectory():
 def __generate_password(length=40):
     return str(uuid.uuid4()) + "-" + str(uuid.uuid4())
 
+def __shorten_computer_name(computer_name, gce_instance):    
+    # Look for MIG related metadata
+    has_metadata = ("metadata" in gce_instance.keys() and
+                    "items" in gce_instance["metadata"])
+    if has_metadata:
+      metadata_created_by = next((x for x in gce_instance["metadata"]["items"] if x["key"] == "created-by"), None)
```

Comment:
You're using 2-space indent here while the rest of the script uses 4-space indent

Reply:
done